src/machines/java/com/enderio/machines/common/blockentity/ZombieGeneratorBlockEntity.java

```diff
@@ -0,0 +1,146 @@
+package com.enderio.machines.common.blockentity;
+
+import com.enderio.api.capacitor.CapacitorModifier;
+import com.enderio.api.capacitor.QuadraticScalable;
+import com.enderio.core.common.sync.FluidStackDataSlot;
+import com.enderio.core.common.sync.SyncMode;
+import com.enderio.machines.common.blockentity.base.PowerGeneratingMachineEntity;
+import com.enderio.machines.common.io.fluid.MachineFluidHandler;
+import com.enderio.machines.common.io.item.MachineInventoryLayout;
+import com.enderio.machines.common.menu.ZombieGeneratorMenu;
+import net.minecraft.core.BlockPos;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.level.block.entity.BlockEntityType;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.material.Fluid;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.capability.IFluidHandler;
+import net.minecraftforge.fluids.capability.templates.FluidTank;
+import org.jetbrains.annotations.Nullable;
+
+import static com.enderio.base.common.init.EIOFluids.DEW_OF_THE_VOID;
+import static com.enderio.base.common.init.EIOFluids.NUTRIENT_DISTILLATION;
+
+public class ZombieGeneratorBlockEntity extends PowerGeneratingMachineEntity {
+
+    private final FluidTank fluidTank;
+
+    private final MachineFluidHandler fluidHandler;
+
+    private final int MINIMUM_FLUID_AMOUNT = 1400;
+
+    public ZombieGeneratorBlockEntity(QuadraticScalable capacity, QuadraticScalable transfer, QuadraticScalable use, BlockEntityType<?> type,
+        BlockPos worldPosition, BlockState blockState) {
+        super(capacity, transfer, use, type, worldPosition, blockState);
+        fluidTank = new FluidTank(10000) {
```

Comment:
Could we store this 10,000 as a constant in this file or a config value? This goes for any of the magic numbers in here :)